src/main/java/uk/ac/cam/cl/dtg/segue/api/AdminFacade.java

```diff
@@ -768,8 +758,6 @@ public Response findUsers(@Context final HttpServletRequest httpServletRequest,
             @QueryParam("id") final Long userId, @QueryParam("email") @Nullable final String email,
             @QueryParam("familyName") @Nullable final String familyName, @QueryParam("role") @Nullable final Role role,
             @QueryParam("schoolOther") @Nullable final String schoolOther,
-            @QueryParam("postcode") @Nullable final String postcode,
-            @QueryParam("postcodeRadius") @Nullable final String postcodeRadius,
```

Comment:
This is a slight feature change - admin users will no longer be able to search by postal code / radius. Given that IPInfoDB has not been working, this feature is probably already not functional anyway.